kolibri/core/content/static/bloom/bloomplayer.htm

```diff
@@ -0,0 +1,16 @@
+<!-- For this file to work it must have a param named "url" which points the folder containing the bloom book. For example,
+   <iframe src="\bloomplayer.htm?url='https:\\example.com\mybook"></iframe> -->
+
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8" />
+    </head>
+    <body style="background-color: #2e2e2e;">
+        <div id="root"><span style="color: #d65649">Loading Bloom Player...</span></div>
+    <script type="text/javascript" src="bloomPlayer-469de4b6cca7dc5c3e04.min.js"></script></body>
+    <!-- At build time, we insert a script tag pointing at bloomPlayer.min.js but with a hash created at build time,
```

Comment:
This comment seems at odds with the script tag above?

Where is the CSS loaded from?

Reply:
CSS are loaded using the DOMWrapper